src/plugins/intel_cpu/src/nodes/common/tile_broadcast_utils.cpp

```diff
@@ -121,8 +121,9 @@ std::vector<NodeDesc> TileBroadcastCommon::getSupportedConfigs(const Node *node)
         config.inConfs[2].constant(constMap[2]);
         config.inConfs[2].setMemDesc(std::make_shared<CpuBlockedMemoryDesc>(ov::element::i32, node->getInputShapeAtPort(2)));
     }
-
-    config.outConfs.resize(node->getChildEdges().size());
+    //outConfs should be same with ports not edges, The structure is node->port->childEdge.
+    outSize = outSize ? outSize : node->getChildEdges().size();
+    config.outConfs.resize(outSize);
```

Comment:
Why cannot we just write the following: 
```
config.outConfs.resize(node->outputShapes.size());
```

Reply:
Sounds good, let me fix this

Reply:
The reason is that `outputShapes` is a protected member of `Node`. `TileBroadcastCommon` could not access it. We need to pass it as an argument of function `getSupportedConfigs` @dmitry-gorokhov 

Therefore, let me change this to the following directly
```
std::vector<NodeDesc> TileBroadcastCommon::getSupportedConfigs(const Node *node, size_t outSize) {
...
config.outConfs.resize(outSize);
...
```

Reply:
The reason is that outputShapes is a protected member of Node. TileBroadcastCommon could not access it. We need to pass it as an argument of function getSupportedConfigs @dmitry-gorokhov

Therefore, let me change this to the following directly
```
std::vector<NodeDesc> TileBroadcastCommon::getSupportedConfigs(const Node *node, size_t outSize) {
...
config.outConfs.resize(outSize);
...
```